f/common/f_setLocalVars.sqf

```diff
@@ -247,33 +149,16 @@ if (f_param_debugMode == 1) then
 
 f_var_groups = allGroups;
 
-// DEBUG
-if (f_param_debugMode == 1) then
-{
- 	_str_f_var_groups = str f_var_groups;
- 	DEBUG_OUTPUT format ["DEBUG %2: f_var_groups = %1",_str_f_var_groups, SLV_NAME];
-};
+DEBUG_OUTPUT(f_var_groups);
 
 // ====================================================================================
 
 // COMMON VARIABLE: f_var_groups_players
 // We will create an array containing all groups with at least one player.
 
-f_var_groups_players = [];
-{
-	_units = units _x;
-	if ({isPlayer _x} count _units >= 1) then {
-		f_var_groups_players set [count f_var_groups_players,_x];
-	};
-} forEach f_var_groups;
-
-// DEBUG
-if (f_param_debugMode == 1) then
-{
- 	_str_f_var_groups = str f_var_groups;
- 	DEBUG_OUTPUT format ["DEBUG %2: f_var_groups = %1",_str_f_var_groups, SLV_NAME];

```

Comment:
This line previously printed the wrong message: f_var_groups instead of f_var_groups_players